pyproject.toml

```diff
@@ -6,7 +6,7 @@ requires = [
 
 [tool.autopep8]
```

Comment:
all this can be dropped as we do not use it...

Reply:
ah okay. changed it to see if it had anything to do with pep8speaks. I'll remove it in a separate PR